scripts/install_ci.sh

```diff
@@ -15,3 +15,5 @@ go get github.com/axw/gocov/gocov
 go get github.com/AlekSi/gocov-xml
 go get github.com/matm/gocov-html
 go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
+
+go get github.com/securego/gosec/cmd/gosec
```

Comment:
install gosec from Azure pipeline, instead of from lint.sh, to be more friendly to running `make lint` locally